EnglishFilter/sections/specific.txt

```diff
@@ -261,6 +261,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+faupto.com###ads-hJiok
```

Comment:
second part is random. 
Also can be blocked 
`||dogemate.com/assets/img/*_728x90.`

Reply:
Where is the second part? I don't see?

Reply:
Does this work? `faupto.com##div[id^="ads-"]` or `faupto.com##div[id*="ads-"]`

Reply:
better `faupto.com##div[id^="ads-"]`

Reply:
> Where is the second part?

`hJiok`)

Reply:
Ok, will change in master.